src/main/java/com/utilaider/logistics/controller/LogisticsController.java

```diff
@@ -87,4 +115,20 @@ public ModelAndView home(HttpServletRequest request, ModelMap map) {
         }
         return new ModelAndView("index", map);
     }
+
+    @RequestMapping(value = "existEmail/{email}")
+    public @ResponseBody
+    Boolean isExistingEmail(@PathVariable("email") String email) {
+        try {
+            return ownerService.getOwnerByUsername(email) == null;
+        } catch (Exception e) {
+            e.printStackTrace();
+            return false;
+        }
+    }
+
+    @RequestMapping("reg")
```

Comment:
This method is not use any where I am Just Checking Registration jsp designed by you.